src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDate = date.split(fromFormat[3]);
+  const oldDateObject = {};
+  const result = [];
+
+  for (let i = 0; i < 3; i++) {
+    oldDateObject[fromFormat[i]] = oldDate[i];
+  }
+
+  if (oldDateObject['YY'] >= 30) {
+    oldDateObject['YYYY'] = 19 + oldDateObject['YY'];
+  } else if (oldDateObject['YY'] < 30) {
+    oldDateObject['YYYY'] = 20 + oldDateObject['YY'];
+  }
+
+  if (oldDateObject['YYYY']) {
+    oldDateObject['YY'] = oldDateObject['YYYY'].slice(2, 4);
```

Comment:
```suggestion
    oldDateObject['YY'] = oldDateObject['YYYY'].slice(2);
```

You can skip end index.